test/org/traccar/protocol/NavisProtocolDecoderTest.java

```diff
@@ -11,28 +11,59 @@ public void testDecode() throws Exception {
 
         NavisProtocolDecoder decoder = new NavisProtocolDecoder(null);
 
+        // Handshake
         verifyNull(decoder, binary(
                 "404E5443010000007B000000130044342A3E533A383631373835303035323035303739"));
+        verifyNull(decoder, binary(
+                "404E5443010000007B000000130047372A3E533A383631373835303035313236303639"));
 
+        // NTCB F5.2 - Single
         verifyPosition(decoder, binary(
                 "404e5443010000000000000059009adb2a3e54250000000000ff1500040b0a1008291838001200760ee600000000000000000000000f1500040b0a10ac20703fb1aec23f00000000320149668f430000000000000000000000000000000000000000000000f3808080"),
                 position("2016-11-11 21:00:04.000", true, 53.74336, 87.14437));
 
+        // NTCB 5.2 - Array (1 message)
         verifyPositions(decoder, binary(
                 "404e544300000000040000005a00c6812a3e410125e3a60700011705071503011030210c0000fa200910e6000000000000000000000001082106150010ae97643f88a39f3f0000000090001fcc6c450000000000000000000000000000000000000000000000f6808080"));
-
```

Comment:
Please revert unnecessary changes like this.

Reply:
Done